handbook/company/pricing-features-table.yml

```diff
@@ -660,28 +664,8 @@
   waysToUse:
     - description: Support ACME as a protocol for MDM certificate generation. Coming soon (2024-12-31) #customer-rosner
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/15611 
-- industryName: Low-level MDM commands for macOS, iOS/iPadOS*, and Windows (e.g. remote restart)
-  documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-commands
-  tier: Free
-  jamfProHasFeature: appleOnly
-  jamfProtectHasFeature: no
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
-  waysToUse:
-    - description: See a list of the upcoming MDM commands and scripts in unified queue. Coming soon (2024-07-15)
-      moreInfoUrl: https://github.com/fleetdm/fleet/issues/15920
-    - description: MDM commands for iOS/iPadOS are coming soon (2024-07-15).
-- industryName: Native macOS update reminders
-  description: Send low-level MDM commands to tell end users to update their OS.
-  moreInfoUrl: https://developer.apple.com/documentation/devicemanagement/schedule_an_os_update
-  tier: Free
-  jamfProHasFeature: yes
-  jamfProtectHasFeature: no
-  usualDepartment: IT
-  productCategories: [Device management]
-  pricingTableCategories: [Device management]
-- industryName: Zero-touch setup for macOS, iOS/iPadOS*, and Windows
+- industryName: Zero-touch setup
```

Comment:
Simplified to **"Zero-touch setup"** and added OS information to the description/tooltip.